Authentication/SignUp/View/SignupController.swift

```diff
@@ -0,0 +1,230 @@
+//
+//  SignupController.swift
+//  Authentication
+//
+//  Created by Daniela Riesgo on 3/7/16.
+//  Copyright © 2016 Wolox. All rights reserved.
+//
+
+import Foundation
+import ReactiveCocoa
+
+public final class SignupController: UIViewController {
+    
+    private let _viewModel: SignupViewModelType
+    private let _signupViewFactory: () -> SignupViewType
+    private let _delegate: SignupControllerDelegate
+    
+    public lazy var signupView: SignupViewType = self._signupViewFactory()
+    
+    private lazy var _notificationCenter: NSNotificationCenter = NSNotificationCenter.defaultCenter()
```

Comment:
Should we receive it in the initializer? For both controllers to use the same.
 @guidomb